Formula/treepl.rb

```diff
@@ -0,0 +1,56 @@
+class Treepl < Formula
+  # cite Smith_2012: "https://doi.org/10.1093/bioinformatics/bts492"
+  desc "Dating phylogenies with penalized likelihood"
+  homepage "https://github.com/blackrim/treePL/wiki"
+  url "https://github.com/blackrim/treePL/archive/fcddad99f4ecbada67ecd210f12fefa762f0d51c.tar.gz"
+  version "2017.05.23"
+  sha256 "7543dbf3cc174bc4b7a39618267439c11c9cd5d1f18ef5b42f0159f389684370"
+  head "https://github.com/blackrim/treePL.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gcc"
```

Comment:
Please change this line like so and move it below `depends_on "nlopt"`
```ruby
depends_on "gcc" if OS.mac? # needs openmp
```

Reply:
This caused audit failure:

```
brewsci/bio/treepl:
  * C: 14: col 3: dependency "gcc" (line 14) should be put before dependency "nlopt" (line 13)
```

so I will reorder this to make audit happy